app/models/spree/user_decorator.rb

```diff
@@ -1,20 +1,22 @@
 module Spree::UserDecorator
 
   def self.prepended(base)
-    base.after_create :create_mollie_customer
+    base.after_commit :ensure_mollie_customer
```

Comment:
Can you add the _on: [:create, :update]_ clause since we don't want mollie customers to be created when they get destroyed in spree.

Reply:
🤦

Reply:
Fixed